ui/src/chatbot.tsx

```diff
@@ -180,8 +204,20 @@ export const XChatbot = (props: Chatbot) => {
               paddingBottom: msgs?.[idx + 1]?.from_user !== from_user ? rem(0.8) : 0,
               color: from_user ? '$text' : botTextColor
             }} >
-            <span className={clas(css.msg, 'wave-s14')} style={{ padding: content?.includes('\n') ? 12 : 6 }}>
+            <span className={clas(css.msg, 'wave-s14')} style={{ padding: content?.includes('\n') ? 12 : 12, paddingBottom: props.feedback && !from_user ? 30 : 0 }}>
```

Comment:
This line is odd: `content?.includes('\n') ? 12 : 12`. Also, does it look good for both single line and multiline cases?